code/__DEFINES/dcs/signals.dm

```diff
@@ -432,6 +432,14 @@
 #define COMSIG_KB_HOLSTER "keybinding_holster"
 #define COMSIG_KB_UNIQUEACTION "keybinding_uniqueaction"
 
+// Ability adding/removing signals
+#define ACTION_GIVEN "gave an action"		//from base of /datum/action/proc/give_action(): (datum/action)
```

Comment:
these still need underscores

Reply:
Underscores finally added, should be all good:tm: now